privaterelay/views.py

```diff
@@ -117,6 +119,19 @@ def invitation(request):
     return redirect('/')
 
 
+@require_http_methods(['POST'])
+def waitlist(request):
+    email = request.POST.get('email')
```

Comment:
```suggestion
    if not settings.WAITLIST_OPEN:
        raise PermissionDenied
    email = request.POST.get('email')
```
Prevent invite creation when waitlist is not open.